apps/notifications/src/alert/services/deployment/deployment.service.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { faker } from '@faker-js/faker';
+import { ConfigService } from '@nestjs/config';
+import { Test, type TestingModule } from '@nestjs/testing';
+import axios from 'axios';
+
+import { LoggerService } from '@src/common/services/logger.service';
+import type { GlobalEnvConfig } from '@src/config/env.config';
+import { DeploymentService } from './deployment.service';
+
+import { MockProvider } from '@test/mocks/provider.mock';
+import { mockAkashAddress } from '@test/seeders/akash-address.seeder';
+
+type Config = Pick<GlobalEnvConfig, 'RPC_NODE_ENDPOINT'>;
+
+describe(DeploymentService.name, () => {
+  describe('getDeploymentBalance', () => {
+    it('should return the deployment balance', async () => {
+      const { service } = await setup();
+      jest.spyOn(axios, 'get').mockResolvedValue({
```

Comment:
suggestion(non-blocking): would be nice to use `axios.create` or even pass it via DI. using global instance may cause some issues, especially with hot reloads